packages/ui/core-components/src/lib/atoms/query-load/QueryLoad.svelte

```diff
@@ -36,8 +40,8 @@
 	{/if}
 {:else if !_data || (!_data.dataLoaded && !_data.error)}
 	<slot name="skeleton" loaded={_data}>
-		<div class="w-full h-64">
-			<Skeleton />
+		<div class="w-full" style="height: {height}px">
```

Comment:
any reason we cant use tailwind here `h-[{height}px]`

Reply:
Tailwind has issues with dynamic values, I believe since the class isn't loaded ahead of time